docs/new-feature-tutorial.md

```diff
@@ -27,14 +59,16 @@ detailed explanation of how everything works.
 
 **Test your changes:** Once you've run the migration, restart memcached
 on your development server (`/etc/init.d/memcached restart`) and then
-restart `run-dev.py` to avoid interacting with cached objects.
+restart the development server (`./tools/run-dev.py`) to avoid interacting
```

Comment:
We might want to just make this link to somewhere else in the docs where we explain how to do this, since you also need to `ctrl-C` here.